### PR TITLE
Properly count InnerSlide children in componentWillReceiveProps

### PR DESCRIPTION
**Pull request by HriBB**
Explanation [here](https://github.com/akiran/react-slick/pull/268#issuecomment-275946140)

### CODE REVIEW

File: src/inner-slider.js [hunk 1]
```diff
@@ -88,7 +88,7 @@ export var InnerSlider = React.createClass({
           index: nextProps.slickGoTo,
           currentSlide: this.state.currentSlide
       });
-    } else if (this.state.currentSlide >= nextProps.children.length) {
+    } else if (this.state.currentSlide >= React.Children.count(nextProps.children)) {
```
Comment (TLib) [inline src/inner-slider.js]: Hi. What is with the "nextProps.children.length" in the changeSlide Method (line 95) ? It should be fixed, too.

**Comment by HriBB** [comment]
@TLib I updated the code and merged master. Please review

**Comment by HriBB** [comment]
@akiran can you please review this PR. Thanks ;)

**Comment by laveesingh** [comment]
Changes have been taken into account and will be reflected soon.
Thanks for your efforts.

**Pull request closed without merging**